Localization/en-US/Mods.CalamityMod.Projectiles.Melee.hjson

```diff
@@ -302,3 +302,4 @@ WulfrumScrewdriverProj.DisplayName: Wulfrum Screwdriver
 YateveoBloomProj.DisplayName: Yateveo Bloom
 YinYoDark.DisplayName: Dark Shard
 YinYoLight.DisplayName: Light Shard
+CosmicShivTrail.DisplayName: Cosmic Shiv Trail
```

Comment:
auto generated translation for content that doesn't have a localisation name. forgot to blacklist when pushing/making pr

Reply:
Can you revert if possible? I can do it manually if it's an inconvenience, but you should be able to take care of it since further changes are already necessary.

Reply:
Yeah I'll take care of it